### PR TITLE
EPMDJ-9099 Support java 17

### PR DESCRIPTION
**Pull request by Mazel-Tovr**

### CODE REVIEW

File: admin-part/src/main/kotlin/jvm/Ext.kt [hunk 1]
```diff
@@ -57,12 +57,13 @@ private fun LambdaParser.checksum(
...
 internal fun JavaClass.parsedBootstrapMethods() = bootstrapMethods()
-    .map { bootstrapMethod ->
-        val (_, lambdaMethodIndex, _) = bootstrapMethod.bootstrapArguments
-        lambdaMethodIndex
-    }.mapNotNull { getConstant(it) as? ConstantMethodHandle }
-    .map { getConstant(it.referenceIndex) as ConstantCP }
-    .map {
+    .mapNotNull { bootstrapMethod ->  // TODO This is hot fix changes to avoid exception on java 17
+        bootstrapMethod.bootstrapArguments.firstOrNull {
+            getConstant(it) as? ConstantMethodHandle != null
```
Comment (ViktoriiaLebedeva) [inline admin-part/src/main/kotlin/jvm/Ext.kt:62]: is

Comment (Mazel-Tovr) [reply admin-part/src/main/kotlin/jvm/Ext.kt]: exactly, ty

File: admin-part/src/main/kotlin/jvm/Ext.kt [hunk 2]
```diff
@@ -57,12 +57,13 @@ private fun LambdaParser.checksum(
  * For more info: https://blogs.oracle.com/javamagazine/post/behind-the-scenes-how-do-lambda-expressions-really-work-in-java
  */
 internal fun JavaClass.parsedBootstrapMethods() = bootstrapMethods()
-    .map { bootstrapMethod ->
-        val (_, lambdaMethodIndex, _) = bootstrapMethod.bootstrapArguments
-        lambdaMethodIndex
-    }.mapNotNull { getConstant(it) as? ConstantMethodHandle }
-    .map { getConstant(it.referenceIndex) as ConstantCP }
-    .map {
+    .mapNotNull { bootstrapMethod ->  // TODO This is hot fix changes to avoid exception on java 17
```
Comment (LikhanovMaxim) [inline admin-part/src/main/kotlin/jvm/Ext.kt:60]: add jira task

File: load-tests/build.gradle.kts [hunk 3]
```diff
@@ -61,7 +61,7 @@ dependencies {
     testImplementation("org.jetbrains.xodus:xodus-entity-store")
 
     testImplementation("org.jacoco:org.jacoco.core")
-    testImplementation("org.apache.bcel:bcel:6.3.1")
+    testImplementation("org.apache.bcel:bcel")
     testImplementation("io.vavr:vavr-kotlin:0.10.0") //TODO remove
```
Comment (LikhanovMaxim) [inline load-tests/build.gradle.kts:65]: add tech debt

**Pull request merged**